src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String STR_PL = " sq.units, ";
+    private static final String STR_UN = " unit, ";
+    private static final String STR_CL = " color: ";
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getInfoFigure() {
+        double area = Math.PI * radius * radius;
```

Comment:
move area calculation to method getArea(), and create interface for this, like Drawable